src/script.c

```diff
@@ -501,3 +504,36 @@ void InitRamScript_NoObjectEvent(u8 *script, u16 scriptSize)
     InitRamScript(script, scriptSize, MAP_GROUP(UNDEFINED), MAP_NUM(UNDEFINED), NO_OBJECT);
 #endif //FREE_MYSTERY_EVENT_BUFFERS
 }
+
+#define WALK_AWAY_SIGNPOST_FRAMES 6
+
+void SetWalkingIntoSignVars(void)
+{
+    gWalkAwayFromSignpostTimer = WALK_AWAY_SIGNPOST_FRAMES;
+    sMsgBoxIsCancelable = TRUE;
+}
```

Comment:
This function gets called exactly once, so this one too has no real reason to exist

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5044/commits/4387b6ef708b88cffbe4660e7df6574cfabd6409